target/linux/ramips/dts/mt7620a.dtsi

```diff
@@ -627,7 +627,7 @@
 	};
 
 	wmac: wmac@10180000 {
-		compatible = "ralink,rt7620-wmac", "ralink,rt2880-wmac";
```

Comment:
even though the first compatibles are completely unused, wouldn't it make sense to change those as well?